### DISCUSSION TITLE
Should we omit `/` route from request tracking by default?

### DISCUSSION THREAD

**Discussion by tomkerkhove** [arcus-azure/arcus.templates#422] [Ideas]
Should we omit `/` route from request tracking by default?

This is because we never use them for APIs and Azure uses it to probe instances of App Service.
This costs customers money because it will track them as requests in Application Insights.

**Comment by pim-simons** [comment]
Just checked and everywhere I used the Arcus HTTP Triggered Function the routePrefix `api` is used for the API's, so as far as I can check `/` is indeed never used.

However I don;t see any of the tracked requests of `/` in Application Insights, do you have an example of that because maybe I am overlooking something.

**Reply by tomkerkhove** [reply]
You will only see this with App Service , not Functions AFAIK

**Reply by stijnmoreels** [reply]
@fgheysels , you have any input on this?
I think we can omit this by default, but we should really document this, and maybe have something to also remove it.

**Reply by fgheysels** [reply]
Yes, I'd say we omit it.

**Reply by stijnmoreels** [reply]
Whooray! 🥳 Let's create an issue out of this.
